DevEdu.API/DevEdu.Business/Services/AuthenticationService.cs

```diff
@@ -1,23 +1,47 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
 using System.Security.Cryptography;
-using System.Text;
-using System.Threading.Tasks;
+using DevEdu.Business.Configuration;
+using DevEdu.DAL.Models;
+using Microsoft.IdentityModel.Tokens;
 
 namespace DevEdu.Business.Services
 {
-    public class AuthenticationService
+    public class AuthenticationService : IAuthenticationService
     {
-        public byte[] GetSalt()
+        private readonly IUserService _userService;
+        public AuthenticationService(){}
```

Comment:
default constructor must be removed

Reply:
I did it for using in userService, thanks for advice.